src/collective/ifttt/actions/ifttt.py

```diff
@@ -88,25 +91,35 @@ class IftttTriggerActionExecutor(object):
     """
     timeout = 120
 
-    def __init__(self, element):
+    def __init__(self, element, context, event):
         self.element = element
+        self.context = context
+        self.event = event
 
     def __call__(self, *args, **kwargs):
-        r = self.element.request
-        ifttt_name = self.element.ifttt_name
+        Title = self.element.Title
+        URL = self.element.URL
```

Comment:
Title and URL should be read from the content rule action event context

```
title = self.context.title
url = self.context.absolute_url()
```

or maybe

```
title = self.context.Title().decode('utf-8', 'ignore')
url = self.context.absolute_url()
```

``Title`` accessor is old enough Plone API that all possible content should support it (independently on where the title is actually stored), but it returns utf-8 encoded version of the title. That should be decoded back to unicode to be usable in JSON. ('ignore' argument for decoding is just defensive extra for ignoring any possible letters with encoding errors).